### DISCUSSION TITLE
How to switch Browser while using NFS for storage?

### DISCUSSION THREAD

**Discussion by ovizii** [mayneyao/eidos#121] [Q&A] [answered]
I am using Win 11 and Chrome and visited eidos.space, entered my serial key, created a space and switched to NFS.

Now I would like to test eidos with Firefox, so I visit eidos.space, enter my serial and am asked to create a DB. I can either create a new space or import from zip file. 

I don't see any way to simply point Firefox to the same folder I selected for NFS on Chrome. 

Is this a misconception on my side or am I missing something?

**Comment by mayneyao** [comment, marked as answer]
https://github.com/mayneyao/eidos?tab=readme-ov-file#requirements 

only chromium-based browsers support this feature
